db/migrate/20240222203722_create_notification_policies.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class CreateNotificationPolicies < ActiveRecord::Migration[7.1]
+  def change
+    create_table :notification_policies do |t|
+      t.references :account, null: false, foreign_key: true, index: { unique: true }
+      t.boolean :filter_not_following, null: false, default: false
+      t.boolean :filter_not_followers, null: false, default: false
+      t.boolean :filter_new_accounts, null: false, default: false
+      t.boolean :filter_private_mentions, null: false, default: true
```

Comment:
I think, for existing users, we should be backfilling these with the current `interactions.must_be_following` and `interactions.must_be_following_dm` to not break people's expectations.